src/material-experimental/mdc-menu/_mdc-menu.scss

```diff
@@ -1,18 +1,60 @@
+@import '@material/menu-surface/mixins';
+@import '@material/menu-surface/variables';
+@import '@material/list/mixins';
+@import '@material/list/variables';
+@import '@material/theme/functions';
 @import '../mdc-helpers/mdc-helpers';
 
 @mixin mat-menu-theme-mdc($theme) {
   @include mat-using-mdc-theme($theme) {
-    // TODO: MDC theme styles here.
+    @include mdc-menu-surface-core-styles($mat-theme-styles-query);
+    @include mdc-list-without-ripple($mat-theme-styles-query);
+
+    // MDC doesn't appear to have disabled styling for menu
+    // items so we have to grey them out ourselves.
+    .mat-mdc-menu-item[disabled] {
+      &, &::after {
+        @include mdc-theme-prop(color, text-disabled-on-background);
```

Comment:
Shouldn't the color styles live in the theme mixin?

Reply:
This is the theme mixin.

Reply:
So it is. I was thrown off by `-theme` not being in the file name

Reply:
Yeah because of the way our glup logic is right now the partial name needs to match the entry point name